tests/tox.ini

```diff
@@ -0,0 +1,75 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+[tox]
+skipsdist=True
+skip_missing_interpreters = True
+envlist = integration
+
+[vars]
+integration_test_path = {toxinidir}/integration/
+
+[testenv]
+deps =
+    pytest
+    pytest-operator
+setenv =
+    PYTHONPATH = {toxinidir}
+    PYTHONBREAKPOINT=ipdb.set_trace
+passenv =
+    HTTP_PROXY
+    HTTPS_PROXY
+    NO_PROXY
+    PYTHONPATH
+    HOME
+    CHARM_BUILD_DIR
+    MODEL_SETTINGS
+
+[testenv:fmt]
+description = Apply coding style standards to code
+deps =
+    black
+    isort
+commands =
+    isort {[vars]integration_test_path}
+    black {[vars]integration_test_path}
+
+[testenv:lint]
```

Comment:
Should we also add `lint` and `static` to the CI since it's already here?

Reply:
It would cover the integration test only, should we add it?

Reply:
We can either add it as additional checks or just remove the relevant sections from this file.

Reply:
Added the checks :)